File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SilkFabric.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class SilkFabric extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_silk_fabric_name;
        this.idDescription = R.string.item_silk_fabric_description;
        this.idImage = R.drawable.silk_fabric;
        this.price = 12L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SilkFabric.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class SilkFabric : Item() {
    override fun configureProperties() {
        idName = R.string.item_silk_fabric_name
        idDescription = R.string.item_silk_fabric_description
        idImage = R.drawable.silk_fabric
        price = 12L
    }
}
